### DISCUSSION TITLE
Looking for a freelancer

### DISCUSSION THREAD

**Discussion by kamatil-dev** [photopea/photopea#6579] [General]
Hi, i would like to create a woocommerce store with a feature that allow user to upload their design and live preview, but instead of uploading products and select print area for each, i want to upload psd files, they will have a smart object layer, so i want to replace the smart object with the image that user (customer) upload, please have a look at those websites:
https://mockcity.com (they use a tool called photopea, a browser version of photoshop, and they use their api to replace the smart object)
https://mockey.ai/ (they use png images, with transparency)

**Comment by photopea** [comment]
sorry, this is not a place for finding employees.

**Comment by llvee** [comment]
Are you still looking for help? I can integrate the Photopea API for you provided access to it is available(has this been determined?) & that you are responsible for any related costs to using the API which there may be. To continue this discussion if you would like to feel free to tag me in an issue on one of your own non photopea repositories.

**Reply by llvee** [reply]
^ @kamatil-dev
